stylesheets/_component.dropzone.scss

```diff
@@ -93,7 +93,7 @@ $dropzone-icon-rotation: 30deg;
 }
 
 .dropzone__help {
-    color: color(text, help);
```

Comment:
> Search: `color(text, help)`
> 6 results in 4 files 

arguably, the colour value for this token should updated instead, making any/all help text instances AA compliant.

Reply:
General help text is AA compliant. This is when the dropzone is used with `form__control` and therefore has the `form__control` off white bg colour.

Reply:
aha ok, I retract my change request!